migrations/0005_pdf_files.sql

```diff
@@ -13,10 +13,10 @@ create table if not exists pdf_files (
   updated_at datetime default current_timestamp
 );
 
--- create indexes for pdf_files table
-create index if not exists idx_pdf_files_username 
-  on pdf_files(username);
-create index if not exists idx_pdf_files_file_key 
-  on pdf_files(file_key);
-create index if not exists idx_pdf_files_status 
-  on pdf_files(status); 
\ No newline at end of file
+-- create indexes for files table
+create index if not exists idx_files_username 
+  on files(username);
+create index if not exists idx_files_file_key 
+  on files(file_key);
+create index if not exists idx_files_status 
+  on files(status); 
```

Comment:
minor: a few more new lines - if you want to just merge and i'll do a one-off sweep on new lines and configure the formatter to add them